src/java.base/share/classes/jdk/internal/classfile/impl/verifier/VerifierImpl.java

```diff
@@ -142,11 +138,8 @@ public static List<VerifyError> verify(ClassModel classModel, ClassHierarchyReso
     }
 
     public static boolean is_eligible_for_verification(VerificationWrapper klass) {
-        String name = klass.thisClassName();
-        return !java_lang_Object.equals(name) &&
-                !java_lang_Class.equals(name) &&
-                !java_lang_String.equals(name) &&
-                !java_lang_Throwable.equals(name);
+        // 8330606 Not applicable here
+        return true;
     }
```

Comment:
It’s good that `ClassFile::verify(…)` will now verify these types.

- https://github.com/openjdk/jdk/pull/19826#pullrequestreview-2382543361